jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -181,6 +181,8 @@ public synchronized void start() throws LifecycleException {
 				this.startupWatchdir();
 
 				this.startupSystem();
+				
+				this.registerMBeans();
```

Comment:
Why? How is it related to InfluxDB?

Reply:
Oh, it's a commit pushed in the wrong branch. It's not related to InfluxDB, but it fixes a bug : when you modify/delete a file in the config folder, jmxtrans reloads the config. But when it do that, it tries to register some MBeans alerady registered, and the process stops without restarting the server jobs.

How can I cancel this commit on github?

Reply:
Ok, it's now cancelled.